configure.ac

```diff
@@ -762,6 +762,29 @@ AC_LINK_IFELSE([AC_LANG_SOURCE([
 )
 LDFLAGS="$TEMP_LDFLAGS"
 
+TEMP_LDFLAGS="$LDFLAGS"
+LDFLAGS="$TEMP_LDFLAGS -mpower8-vector"
```

Comment:
Shouldn't this be CPPFLAGS or CXXFLAGS?

Reply:
Agree, but this should be assigned to POWER_VSX_CXXFLAGS or so above, in an AX_CHECK_LINK_FLAG test.